advocacy_docs/partner_docs/SIBVisionsGuide/05-UsingVisionX.mdx

```diff
@@ -0,0 +1,477 @@
+---
+title: 'Using VisionX'
+description: 'Walkthroughs of example usage scenarios'
+---
+
+This section shows the following VisionX usage scenarios with EDB Postgres Advanced Server and EDB Postgres Extended.
+
+**Semi-automated migration of an Oracle Forms application / Oracle APEX to an EDB Postgres / VisionX Low Code application:**
+- Oracle Forms migration based on the FMB file.
+- Oracle Forms / APEX migration based on the data model
+
+**Use the VisionX Low-code platform to visually build web and mobile applications that use an EDB Postgres Advanced Server/EDB Postgres Extended instance as a data source:**
+- Create a Screen that connects to an existing table from an EDB Postgres Advanced Server/EDB Postgres Extended instance.
+- Create a Screen that creates a table in the connected EDB Postgres Advanced Server/EDB Postgres Extended instance.
+
+## FMB-based migration of an Oracle Forms application to an EDB Postgres / VisionX Low Code application
+
+This example shows how an Oracle Forms application is migrated to a VisionX low-code application using an FMB file (= Oracle Forms source file). The first step is to migrate the application. In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended using the usual tools such as EDBs MTK.
```

Comment:
Ask Kashif - WE don't use MTK to migrate to Postgres Extended.  So don't think this fits.